src/main/java/com/ucsb/demonextjsspringtodoapp/services/Auth0MembershipService.java

```diff
@@ -0,0 +1,76 @@
+package com.ucsb.demonextjsspringtodoapp.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.ucsb.demonextjsspringtodoapp.repositories.AdminRepository;
+
+/**
+ * Service object that determines whether a user is a member of the google org or not.
+ */
+@Service
+public class Auth0MembershipService implements MembershipService {
+
+  private Logger logger = LoggerFactory.getLogger(Auth0MembershipService.class);
+
+
+  @Value("${app.member.hosted-domain}")
```

Comment:
I'm getting the error `Could not resolve placeholder 'app.member.hosted-domain' in value "${app.member.hosted-domain}"` when trying to deploy in heroku.

Reply:
So I'm going to resolve this by adding it to the `secrets-heroku.properties.SAMPLE` file.